### ISSUE TITLE
[web] Live area error in a11y mode in Desktop

### ISSUE THREAD

**Issue by nturgut** [flutter/flutter#45204]
Live areas (such as a toast, dialog) does not receive the a11y focus. The information in those fields will be either impossible (if the toast disappears) or very hard for the assistive technology user to understand.

Tested on: Safari, MacOS, VoiceOver

**Comment by csells** [comment]
@nturgut can you provide a repro project that demonstrates the issue?

**Comment by nturgut** [comment]
Thanks for spending time on the web issues :) 

It is not necessary. It fails in any live area on Safari MacOS Voice over, for example Gallery.

It is mostly because I'll be focusing desktop a11y bugs later in the year.

You can think "Flutter Web a11y" project as a todo list I keep. Please feel free to add more a11y desktop bugs if you see any.
